pubspec.yaml

```
@@ -0,0 +1,8 @@
+name: jesi_dart_analysis
+description: A package providing lint rules for Dart and Flutter which are used at JESI
+version: 1.0.0
+homepage: https://github.com/jesims/jesi_dart_analysis
+publish_to: none
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
```

Can use caret versioning, and adding dev dependencies means the IDE can find the files when used in `include:`
```suggestion
  sdk: '^2.12.0'

dev_dependencies:
  lints: '^1.0.1'
  flutter_lints: '^1.0.0'
```

---

Sure.